shared/src/main/scala/Context.scala

```diff
@@ -88,6 +88,8 @@ class Context private (
     if useStack then getTopCxt().push(items*)
     else stack ++= items
 
+  def length(): Int = stack.length
```

Comment:
```suggestion
  def length: Int = stack.length
```
It's totally a matter of personal preference, but since this is Scala, figured we could do it like this. It's a constant time operation anyway